src/content/docs/vulnerability-management/integrations/aws.mdx

```diff
@@ -17,12 +17,12 @@ Optional:
 
 ## 1. Create API destination
 
-1. In your AWS UI navigate to **EventBridge → Integrations → API destinations → Create API destination**.
+1. In your AWS UI navigate to **EventBridge > Integrations > API destinations > Create API destination**.
 2. Fill in the prompts.
 3. Construct your endpoint using the pattern below, and enter it into API destination endpoint:
 
 ```
-https://security-ingest-processor.service.newrelic.com/v1/security/webhooks/awssecurityhub?Api-Key=<var><NEW RELIC LICENSE KEY></var>
+https://security-ingest-processor.service.newrelic.com/v1/security/webhooks/awssecurityhub?Api-Key=<var><NEW_RELIC_LICENSE_KEY></var>
```

Comment:
```suggestion
https://security-ingest-processor.service.newrelic.com/v1/security/webhooks/awssecurityhub?Api-Key=<var>NEW_RELIC_LICENSE_KEY</var>
```

User-supplied values should not be wrapped in any particular punctuation unless that punctuation is literally required, to avoid ambiguity about what the user-supplied value should be formatted like.